src/convert.ts

```diff
@@ -384,6 +455,62 @@ function entityTypeUpdate(prefix: string, entitySet: EntitySet, parentTypes: Arr
   };
 }
 
+function entityFunctionImportPath(entitySet: EntitySet, functionImport:FunctionImport): PathItem {
+  const pathItem = {};
+  let operation = {}
+  if(`${functionImport.returnType}`.startsWith("Collection(")){
+    const returnType =`${functionImport.returnType}`.slice(11,-1)
+
+    operation = {
+      operationId: functionImport.name,
+      summary: functionImport.label,
+      parameters: functionImport.parameters,
+      responses: {
+        '200': {
+          description: `${functionImport.returnType}`,
+          schema: {
+            type: 'object',
+            properties: {
+              value: {
+                type: 'array',
+                items: {
+                  $ref: `#/definitions/${returnType}`
+                }
+              }
+            }
+          }
+        },
+        default: defaultResponse
+      }
+    };
+    if(instanceOfSAPEntitySet(entitySet)){
+        //OData v2 (SAP) uses d instead of value as the out response element
+      //Ref https://www.odata.org/documentation/odata-version-2-0/json-format/
+      operation['responses']['200']['schema']['properties']['d'] = operation['responses']['200']['schema']['properties']['value'];
```

Comment:
In the SAP OData serivce I'm consuming the envelop is not using `value` but `results` see #44 
Not sure why.

Another "issue" is that this `d` won't have `type: 'object'` assigned, not sure what impact this has on the OpenApi spec